src/commands/ScheduleSend.ts

```diff
@@ -44,7 +43,7 @@ export default class ScheduleSend extends Command {
      * Pull the user input values from the command
      */
     const message = interaction.options.getString('message', true);
-    const member = interaction.member;
+    const member = interaction.member ? interaction.member : '';
```

Comment:
JavaScript has a special operator (??) to reduce this type of redundancy that returns the right side if the left side variable is `null` or `undefined`